.pre-commit-config.yaml

```diff
@@ -0,0 +1,51 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.5.0
```

Comment:
Just to make sure that these rev numbers are up to date. Run

`precommit autoupdate` and it will automatically update the versions in this file for us.